packages/main/src/Avatar.js

```diff
@@ -208,6 +208,35 @@ const metadata = {
 			propertyName: "image",
 			type: HTMLElement,
 		},
+		/**
+		 * Defines the optional <code>ui5-badge</code> that will be used for visual affordance.
```

Comment:
I suggest only using the term "badge", because the idea of the slot is to make it generic, so let's sound generic:
"Defines the optional **badge** that will be used for visual affordance."

Then, you can add a note that `ui5-badge` component is our recommended offering.

Reply:
Done